src/main/resources/webapp/partials/modals/visitRescheduleModal.html

```diff
@@ -21,6 +21,16 @@ <h4 class="modal-title">{{rescheduleModalHeader}}</h4>
                         <input id="startTimeInput" type="text" class="form-control" mds-time-picker placeholder="{{msg('prevac.startTimePlaceholder')}}" ng-model="form.dto.startTime">
```

Comment:
if you are hiding or disabling planned date input you should do the same for planned date time input too

Reply:
Right, I thought that it could be useful but after rethinking it now I agree with you.